front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -48,20 +48,18 @@ const StyledIdeasShow = styled(IdeasShow)<{ hasNextPreviousControl?: boolean }>`
   padding-right: 60px;
 
   ${({ theme }) => media.tablet`
-    margin-top: ${(props) =>
-      props.hasNextPreviousControl ? '0' : theme.menuHeight}px;
     min-height: calc(100vh - ${theme.mobileTopBarHeight}px);
     padding-top: 35px;
   `}
 
   ${media.phone`
-    padding-top: ${(props) => (props.hasNextPreviousControl ? '0px' : '25px')};
     padding-left: 15px;
```

Comment:
Since next/previous controls have been removed pending a new solution, I've removed this styling since it was indeed confusing with the parent container's padding :+1: